src/libs/SearchUIUtils.ts

```diff
@@ -518,8 +518,7 @@ function getReportSections(data: OnyxTypes.SearchResults['data'], metadata: Onyx
         }
     }
 
-    // Filter out reports with no transactions to prevent the wrong number of the selected options
-    return Object.values(reportIDToTransactions).filter((report) => report.transactions.length);
+    return Object.values(reportIDToTransactions);
```

Comment:
Ahhh so many fond memories of many many hours spent catching these pesky reports without any transactions.
Remember @luacmartins? :D 

I'm quite happy we're now making "empty" reports a first class citizen 👍